src/message/index.js

```diff
@@ -8,17 +8,23 @@ const jsonPretty = (data) => JSON.stringify(data, undefined, 2);
 const postMessage = async () => {
   try {
     const token = context.getRequired("slack-bot-user-oauth-access-token");
-    const channel = context.getRequired("slack-channel");
+    const channels = context.getRequired("slack-channel");
     const text = context.getRequired("slack-text");
-
-    const payload = buildMessage(channel, text, optional());
-
-    context.debug("Post Message PAYLOAD", payload);
-    const result = await apiPostMessage(token, payload);
-    context.debug("Post Message RESULT", result);
-
-    const resultAsJson = jsonPretty(result);
-    context.setOutput("slack-result", resultAsJson);
+    const resultsAsJson = []
+    
+    for(let channel of channels.split(',')) {
+      channel = channel.trim()
+      
+      const payload = buildMessage(channel, text, optional());
+
+      context.debug("Post Message PAYLOAD", payload);
+      const result = await apiPostMessage(token, payload);
+      context.debug("Post Message RESULT", result);
+
+      resultsAsJson.push(jsonPretty(result))
+    }
+    
+    context.setOutput("slack-result", resultsAsJson);
```

Comment:
This would break the backward compatibility, since the result would now be an array.

To keep safe, we should probably add array[0] to "slack-result" and the entire array to a new "slack-results"

Thoughts?